examples/step-47/doc/results.dox

```diff
@@ -211,3 +211,85 @@ make sense:
   addition should not be overly difficult using, for example, the
   FEInterfaceValues class combined with MeshWorker::mesh_loop() in the
   same spirit as we used for the assembly of the linear system.
+
+
+  <h4> Derivation for the simply supported plates </h3>
```

Comment:
What the CI is still complaining about is this line:
```suggestion
  <h4> Derivation for the simply supported plates </h4>
```

Reply:
Thanks!